db/migrate/20221021084927_create_community_stories.rb

```diff
@@ -0,0 +1,23 @@
+class CreateCommunityStories < ActiveRecord::Migration[7.0]
+  def change
+    create_table :community_stories do |t|
+      t.belongs_to :interviewer, null: false, foreign_key: {to_table: :users}
+      t.belongs_to :interviewee, null: false, foreign_key: {to_table: :users}
+
+      t.string :uuid, null: false
```

Comment:
```suggestion
      t.string :uuid, null: false, index: {unique: true}
```

(Pretty sure this works, rather than having it as a seperate line below)